apps/desafio_api/lib/account/schemas/account.ex

```diff
@@ -0,0 +1,22 @@
+defmodule DesafioApi.Account.Schemas.Account do
+  @moduledoc """
+    Account entity schema
```

Comment:
Como aqui segue a sintaxe de markdown deixar um espaço no começo pode fazer diferença
```suggestion
  Account entity schema
```